OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/settings/ApplicationSettings.java

```diff
@@ -95,6 +95,7 @@ private void setSettingValue(Setting setting, Object value) {
         DROPBOX_TOKEN( "dropbox.token", "", SettingType.STRING ),
         LATEX_PATH( "latex.path", "", SettingType.STRING ),
         USER_LASTDIR( "user.lastdir", "", SettingType.STRING ),
+        USER_LASTFILE( "user.lastfile", "/openlatexstudio/welcome.tex", SettingType.STRING ),
```

Comment:
It doesn't seem to make sense to keep this default here. welcome.tex is a classpath resource, not user's file, thus will never qualify as a "File".